### PR TITLE
Add debugging info for foreman_openscap

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: plugins/foreman_openscap/0.7/index.md [hunk 1]
```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
...
+    1:
+      :profile: 'xccdf_org.ssgproject.content_profile_stig-java-upstream'
+      :content_path: '/var/lib/openscap/content/fe93f99c14251cc76e92b9da71c351c8ba45fbd3639a2cd55911ef6f7db1b650.xml'
+      # Download path
+      # A path to download SCAP content from proxy
+      :download_path: '/compliance/policies/1/content/fe93f99c14251cc76e92b9da71c351c8ba45fbd3639a2cd55911ef6f7db1b650'
+      :tailoring_path: ''
+      :tailoring_download_path: ''
+
+There will be an entry for each policy that is assigned to your host.
```
Comment (ares) [inline plugins/foreman_openscap/0.7/index.md]: The policy entry starts by number followed by colon. Policy attributes are indented by 2 spaces.

File: plugins/foreman_openscap/0.7/index.md [hunk 2]
```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
...
+
+
+* Running scans manually
+
+You can try running foreman_scap_client manually by executing
+
+    > foreman_scap_client $policy_id
```
Comment (ares) [inline plugins/foreman_openscap/0.7/index.md]: I'd remove `>`, usually `$` is used for commands that should be executed under normal user, `#` for commands ran under root. In this case I think we can simply remove the character completely.

File: plugins/foreman_openscap/0.7/index.md [hunk 3]
```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
...
+
+from your command line, where `$policy_id` is a policy id from config file (1 in case of example config file above)
+
+
+* Information in logs
+
+If running scan manually succeeds and there are no errors, try switching logging to DEBUG on proxy with openscap feature that your host uploads reports to and restart the proxy. Then use
+
+    > tail -f /var/log/foreman-proxy/proxy.log
```
Comment (ares) [inline plugins/foreman_openscap/0.7/index.md]: I'd remove `>`, usually `$` is used for commands that should be executed under normal user, `#` for commands ran under root. In this case I think we can simply remove the character completely.

File: plugins/foreman_openscap/0.7/index.md [hunk 4]
```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
...
+
+
+* Running with Katello
+
+When running with [Katello](/plugins/katello/index.html), make sure your host is registered as [Content Host](plugins/katello/nightly/user_guide/content_hosts/index.html). Consumer certificates are used for client authentication instead of Puppet certs.
+
+
+* Scans based on cronlines
+
+Scanning and report generation are based on cron. You can inspect all the cron lines in ```/etc/cron.d/foreman_scap_client_cron```. This file is managed by Puppet and any changes will be overwritten on next Puppet run.
```
Comment (ares) [inline plugins/foreman_openscap/0.7/index.md]: s/and any changes/and any manual changes/
s/next Puppet run/next Puppet agent run/

File: plugins/foreman_openscap/0.7/index.md [hunk 5]
```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
...
+
+
+* Config for client
+
+Config file for foreman_scap_client is located at ```/etc/foreman_scap_client/config.yaml```. It should look something like this:
```
Comment (ares) [inline plugins/foreman_openscap/0.7/index.md:363]: This file is also managed by Puppet and any manual changes will be rewritten on next Puppet agent run.

**Review by ares — CHANGES_REQUESTED** [review]
Very nice write-up. I left few possibly improving comments.

**Comment by xprazak2** [comment]
I made changes as suggested.

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**